client/my-sites/checkout/composite-checkout/components/item-variation-picker/placeholder.tsx

```diff
@@ -0,0 +1,43 @@
+import { keyframes } from '@emotion/react';
```

Comment:
This whole component is part of a temporary fix and can be removed once the A/B test is complete.